From: Alec Leamas <leamas.alec@gmail.com>
Date: Sun, 19 Jul 2020 17:42:01 +0200
Subject: [PATCH] build: Handle missing doc files.

A simple pointer added since the original documentation is
removed in the package. Users can download the docs from
locations specified in this file.

Not acceptable upstream.

Forwarded: not-needed
---
 CMakeLists.txt     |  8 ++++----
 data/help_web.html | 51 +++++++++++++++++++++++++++++++++++++++++++++++++++
 2 files changed, 55 insertions(+), 4 deletions(-)
 create mode 100644 data/help_web.html

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 628e06d..a72e629 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1994,7 +1994,7 @@ if (APPLE)
   )
 
   set_source_files_properties(
-    data/doc/help_web.html
+    data/help_web.html
     PROPERTIES
       MACOSX_PACKAGE_LOCATION SharedSupport/doc
   )
@@ -2051,12 +2051,12 @@ if (APPLE)
   if (OCPN_BUNDLE_DOCS )
     set(
       DOC_FILES
-      data/doc/help_web.html
+      data/help_web.html
       data/doc/help_en_US.html
       ${WIKI_DOC_FILES}
     )
   else (OCPN_BUNDLE_DOCS )
-    set(DOC_FILES data/doc/help_web.html)
+    set(DOC_FILES data/help_web.html)
   endif (OCPN_BUNDLE_DOCS )
 
   target_sources(
@@ -2207,7 +2207,7 @@ if (NOT APPLE)
   if (OCPN_BUNDLE_DOCS )
     install(DIRECTORY data/doc/ DESTINATION ${PREFIX_PKGDATA}/doc)
   else (OCPN_BUNDLE_DOCS )
-    install(FILES data/doc/help_web.html DESTINATION ${PREFIX_PKGDATA}/doc)
+    install(FILES data/help_web.html DESTINATION ${PREFIX_PKGDATA}/doc)
   endif (OCPN_BUNDLE_DOCS )
 
   if (UNIX)
diff --git a/data/help_web.html b/data/help_web.html
new file mode 100644
index 0000000..e71cf85
--- /dev/null
+++ b/data/help_web.html
@@ -0,0 +1,51 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+
+  <meta content="text/html; charset=ISO-8859-1" http-equiv="content-type">
+  <title>index</title>
+
+
+</head>
+
+
+<body style="background-color: rgb(255, 255, 204);">
+
+<br>
+
+<br>
+
+<br>
+
+<br>
+
+<br>
+
+<br>
+
+<br>
+
+<br>
+
+<br>
+
+<br>
+
+<br>
+
+<br>
+
+<br>
+
+<br>
+
+<div style="text-align: center;"><span style="color: rgb(51, 51, 51);">For Online Help</span><br style="color: rgb(51, 51, 51);">
+
+<span style="color: rgb(51, 51, 51);">click on</span><br>
+
+<a href="https://opencpn.org/">OpenCPN.org</a><br>
+
+</div>
+
+</body>
+</html>
